Plugin/PaymentVaultDeleteToken.php

```diff
@@ -60,14 +49,11 @@ public function __construct(
      * @return PaymentTokenInterface[]|void
      * @throws NoSuchEntityException
      */
-    public function beforeDelete(
-        PaymentTokenRepositoryInterface $subject,
-        PaymentTokenInterface $paymentToken
-    ) {
+    public function beforeDelete(PaymentTokenRepositoryInterface $subject, PaymentTokenInterface $paymentToken) {
```

Comment:
`$subject` parameter is not being used, can we delete it?

Reply:
We can't, because this is in a plugin so it has to match the magento function definition